helpers/Notification.js

```diff
@@ -80,7 +80,7 @@ class Notification {
    *
    * @method newArticle
    * @param {object} requestInfo request object
-   * @param {uuid} articleSlug the id of the article
+   * @param {uuid} articleSlug the slug of the article
```

Comment:
Is articleSlug a `uuid`?

Reply:
Yes, it is a uuid in the database. But when returned in a response object it becomes a string.